test/helpers.js

```diff
@@ -320,3 +320,40 @@ export const itReturnsErrorForEmptyFields = ({
     }),
   );
 };
+
+export const itReturnsErrorForUnverifiedVendor = ({
+  endpoint,
+  method,
+  user,
+  data = {},
+  useExistingUser = false,
+}) => {
+  let token;
+
+  beforeEach(async () => {
+    if (useExistingUser) {
+      const loggedInUser = await loginUser(user);
+      token = loggedInUser.token;
+    } else {
+      token = await addUser({ ...user, vendor: { verified: false } });
```

Comment:
Nice work here. However, it is prefereble to update only the verified key and not the whole vendor object.

```suggestion
      token = await addUser({ ...user, vendor: { ...user.vendor, verified: false } });
```